playbooks/provisioning/aws/provision_aws.yml

```diff
@@ -0,0 +1,16 @@
+- hosts: localhost
+  become: no
+  gather_facts: no
+  pre_tasks:
+  - debug: var=openshift_cluster_id
+  - debug: var=openshift_env_id
+
+  # TODO: properly handle these defaults, currently role defaults are overriding
+  - fail:
+      msg: "Missing required variable: openshift_cluster_id"
+    when: openshift_cluster_id is not defined
```

Comment:
hmm, seems odd, maybe that avoids values that are set in defaults, which is what I'm hitting.